components/runtime/openjdk-11/openjdk.p5m

```diff
@@ -0,0 +1,1011 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2016 Alexander Pyhalov
+# Copyright 2021 Andreas Grueninger, Grueninger GmbH, (grueni). All rights reserved. 
+#
+
+<transform path=usr/jdk/instances/$(OPENJDK_INSTANCE)/lib/.*\.so -> default pkg.linted.userland.action001.2 true>
+
+set name=pkg.fmri	value=pkg:/$(COMPONENT_FMRI)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.summary value="OpenJDK $(COMPONENT_VERSION) Runtime Environment"
+set name=info.classification value="$(COMPONENT_CLASSIFICATION)"
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
+set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+
+license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
+
+dir path=usr/jdk/instances/$(OPENJDK_INSTANCE)
+link path=usr/jdk/$(OPENJDK_INSTANCE) target=instances/$(OPENJDK_INSTANCE)
+
+## as long as the two links are defined in openjdk8 they can't used here
+# link path=usr/jdk/latest target=$(OPENJDK_INSTANCE) pkg.linted.userland.action002.0=true
+# link path=usr/java target=jdk/$(OPENJDK_INSTANCE)
+
```

Comment:
The way to handle this would normally be to create a separate PR which removes this from openjdk-8 (increasing its COMPONENT_REVISION) and add a comment to the PR saying "Merge this PR after the one for openjdk-8".

Reply:
Thinking more about the latest links makes me wonder whether it should also be mediated.

Reply:
Good idea to mediate the two links

Reply:
Not sure. Latest clearly should be 'latest', '/usr/java' once was used by illumos-gate builds. Perhaps, now it can be mediated.

Reply:
It depends on the definition of "latest". From the view point of the repository it should it should be linked to the newest available version. And not be mediated.